fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/explain/NodeInfo.java

```diff
@@ -0,0 +1,191 @@
+    VALUES_DATA(
+            "ValuesData",
+            Icon.IN_MEMORY_TEMPORARY_DATA,
+            "Values",
+            "A list of values as specified."),
+    COVERING_INDEX_SCAN_OPERATOR(
+            "CoveringIndexScanOperator",
+            Icon.DATA_ACCESS_OPERATOR,
+            "Covering Index Scan",
+            "A covering index scan operator uses a secondary index to quickly find records in a given search range without looking up the base record."),
+    INDEX_SCAN_OPERATOR(
+            "IndexScanOperator",
+            Icon.DATA_ACCESS_OPERATOR,
+            "Index Scan",
+            "An index scan operator uses a secondary index to quickly find records in the index within a given search range."),
+    INTERSECTION_OPERATOR(
+            "IntersectionOperator",
+            Icon.COMPUTATION_OPERATOR,
+            "Intersection",
+            "An intersection operator processes its inputs in a common order and only returns those records that appear in all inputs."),
+    NESTED_LOOP_JOIN_OPERATOR(
+            "NestedLoopJoinOperator",
+            Icon.COMPUTATION_OPERATOR,
+            "Nested Loop Join",
+            "A nested loop join operator performs a relational join between its two input operands by reevaluating the right (inner) side for each left (outer) record."),
+    LOAD_BY_KEYS_OPERATOR(
+            "LoadByKeysOperator",
+            Icon.DATA_ACCESS_OPERATOR,
+            "Load By Keys",
+            "A load by keys operator loads records from the database that match the set of given keys."),
+    PREDICATE_FILTER_OPERATOR(
+            "PredicateFilterOperator",
+            Icon.COMPUTATION_OPERATOR,
+            "Predicate Filter",
+            "A predicate filter operator processes the input and returns only those records for which the predicate expression is true."),
+    SCAN_OPERATOR(
+            "ScanOperator",
+            Icon.DATA_ACCESS_OPERATOR,
+            "Scan",
+            "A scan operator loads a set of records from the database that are within the given range of primary keys."),
+    SCORE_FOR_RANK_OPERATOR(
+            "ScoreForRankOperator",
+            Icon.DATA_ACCESS_OPERATOR,
+            "Score For Rank",
+            "A score for rank operator converts ranks to scores and executes its input plan with the conversion results bound in named parameters."),
+    TABLE_FUNCTION_OPERATOR(
+            "TableFunctionOperator",
+            Icon.COMPUTATION_OPERATOR,
+            "Table Function",
+            "A table function operator returns a table of records each time it is evaluated with a set of arguments that are specific to each individual table function."),
+    TEXT_INDEX_SCAN_OPERATOR(
+            "TextIndexScanOperator",
+            Icon.DATA_ACCESS_OPERATOR,
+            "Text Index Scan",
+            "A text index scan operator uses a text index to quickly find records in the index for a set of given tokens."),
```

Comment:
```suggestion
            "A text index scan operator uses a text index to quickly find records in the index for the given full text predicate."),
```
since our full text scans offer more than just "find these tokens".